src/tests/ftest/util/test_utils_container.py

```diff
@@ -910,32 +910,80 @@ def punch_records(self, indices, punch_dkey=True):
         return count
 
     @fail_on(CommandFailure)
-    def get_prop(self, properties=None):
-        """Get container property by calling daos container get-prop.
+    def set_prop(self, *args, **kwargs):
```

Comment:
Don't we need to add descriptions for `*args` and `**kwargs`? Also, what's the difference between `*args` and `**kwargs`?

Reply:
We could add a description, but it's just pass-through to `DaosCommand.container_get_prop`. I.e. the actual arguments to this function are the same arguments to `DaosCommand.container_get_prop`. The benefit to `*args, **kwargs` is that we don't have to update arguments here when `DaosCommand.container_get_prop` arguments are updated, and we ensure the argument names are exactly the same.

`*args` is for positional arguments and `**kwargs` is for named arguments. For example, `fun2` just passes arguments to `fun1`:
```
>>> def fun1(*args, **kwargs):
...   print('args = ', args)
...   print('kwargs = ', kwargs)
...
>>> def fun2(*args, **kwargs):
...   fun1(*args, **kwargs)
...
>>> fun2('pos1', 'pos2', kw1=1, kw2=2)
args =  ('pos1', 'pos2')
kwargs =  {'kw1': 1, 'kw2': 2}
```

Reply:
Or, a more concrete example:
```
>>> def fun1(my_arg):
...   print('my_arg = ', my_arg)
... 
>>> def fun2(*args, **kwargs):
...   fun1(*args, **kwargs)
... 
>>> fun2('foo')
my_arg =  foo
>>> fun2(my_arg='foo')
my_arg =  foo
```

Reply:
Yes, it is better to add a description... Anyone who sees this script may not have a clue what is going on here (unless doing a deep dive in the script)?

Reply:
I was thinking with the comment `See DaosCommand.container_get_prop for arguments.` and an understanding of `*args, **kwargs`, it would make sense. But I guess most people don't encounter that often. I'll update :)

Reply:
I see. Thanks for the explanation!
(As you mentioned, this is useful for a pass-through, but I think it makes it harder to keep track of the arguments.)

Reply:
> it makes it harder to keep track of the arguments

You're right. I did hesitate on that, but eventually I'd like to figure out a simpler architecture overall so we don't have wrappers for wrappers for wrappers where the arguments are duplicated each time.